lib/io/src/ui/confirmation_dialog.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:paintroid/io/src/ui/generic_dialog.dart';
+
+Future<bool?> showConfirmationDialog(BuildContext context) =>
```

Comment:
If this is only used in one widget, please move it there.
If this is used in multiple widgets, please name it more specific, because now it seems like a generic widget is called, but it already uses hardcoded text

Reply:
Renamed the file and the function. This function can be reused later like other dialog functions.